src/rustbot.md

```diff
@@ -0,0 +1,62 @@
+# Mastering @rustbot
+
+`@rustbot` (also known as `triagebot`) is a utility robot that is mostly used to
+allow any contributor to achieve certain tasks that would normally require GitHub
+membership to the `rust-lang` organization. Its most interesting features for
+contributors to `rustc` are isssue claiming and relabeling.
+
+## Issue claiming
+
+`@rustbot` exposes a command that allows anyone to assign an issue to themselves.
+If you see an issue you want to work on, you can send the following message as a
+comment on the issue at hand:
+
+    @rustbot claim
+
+If you are a member of the `rust-lang` organization, `@rustbot` will assign the
+issue to you. Otherwise, `@rustbot` will assign itself as a placeholder and edit
+the top comment to reflect the fact that the issue is now assigned to you.
```

Comment:
I don't think this is true anymore; I think rustbot will assign you regardless of whether you're part of the org. For example, I'm not a member of rust-lang, but rustbot assigns me directly. Cc @Mark-Simulacrum - am I understanding it correctly?

Reply:
I've definitely seen it assign itself before, not the person claiming. Not sure why it assigns you but not others though.

Reply:
Maybe it likes me ;)

I haven't seen it do that in a while though.

Reply:
I just looked and I've seen `@rustbot claim` assign issues to `@lcnr`, who is not a GitHub org member, and `@jumbatm`, who is likewise not an org member. It definitely used to assign itself, but the most recent issue `rustbot` is assigned to was opened July 31.

Reply:
(how is lcnr not an org member? they've implemented like half of min_const_generics 😆 )

Reply:
They are now :P
Btw they were member or the Prioritization WG which is "enough" to be part or the GitHub org

Reply:
IIRC, triagebot will assign you directly if you either are an org member or commented on the issue/PR where the assignment happens. But I would consider this an implementation detail - does it not suffice to say that sometimes you will be assigned directly and sometimes indirectly?

Reply:
Thanks for the clarification! I think you’re right, just saying that you might be assigned indirectly is the best way of presenting things indeed